lib/deep-equal.test.js

```diff
@@ -364,6 +365,40 @@ describe("deepEqual", function() {
         assert.isFalse(checkDeep);
     });
 
+    it("returns false if object has different symbolic properties", function() {
+        var obj1 = {};
+        var obj2 = {};
+        obj1[symbol] = 42;
+        obj2[symbol] = 43;
+        var checkDeep = samsam.deepEqual(obj1, obj2);
+        assert.isFalse(checkDeep);
```

Comment:
MINOR: Can we make the tests a bit more concise?

```js
var obj1 = {[symbol]: 42};
var obj2 = {[symbol]: 43};

assert.isFalse(samsam.deepEqual(obj1, obj2));
```

Reply:
Actually, I tried that first, and the longer complained about an unexpected opening bracket. Changing it to this fixed the linting error.